handlers.go

```diff
@@ -47,31 +49,41 @@ func setSessionCookie(w http.ResponseWriter, id int) {
 }
 
 func signupSubmit(w http.ResponseWriter, r *http.Request) {
+	username := r.FormValue("username")
+	// TODO(@seoyoungcho213): Validate user data way better here.
+	if username == "" {
```

Comment:
i don't think it's needed since it is already handled by SQL. If u wanna double check, please use the function that u created yesterday.

Reply:
This is just a placeholder, as it says in the TODO, you will be responsible for the actual validation logic.